src/node/test/call_test.js

```diff
@@ -188,6 +188,91 @@ describe('call', function() {
       }, TypeError);
     });
   });
+  describe('startBatch with message', function() {
+    it('should fail with non-buffer arguments', function() {
+      var call = new grpc.Call(channel, 'method', getDeadline(1));
+      assert.throws(function() {
```

Comment:
nit: somewhat for consistency with `startBatch with status` test set, create a new call for each of these `TypeError` test cases?